sys/fuchsia/debug.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
```

Comment:
TODO(palmer): Add a link to debug.fidl, and make the `zx_handle`s below into `zx_resource`s. For a follow-up CL.